crates/bevy_gilrs/src/converter.rs

```diff
@@ -40,6 +40,7 @@ pub fn convert_axis(axis: gilrs::Axis) -> Option<GamepadAxisType> {
         // The `axis_dpad_to_button` gilrs filter should filter out all DPadX and DPadY events. If
         // it doesn't then we probably need an entry added to the following repo and an update to
         // GilRs to use the updated database: https://github.com/gabomdq/SDL_GameControllerDB
-        gilrs::Axis::Unknown | gilrs::Axis::DPadX | gilrs::Axis::DPadY => None,
+        gilrs::Axis::Unknown => Some(GamepadAxisType::Other(code.into_u32() as u8)),
```

Comment:
This makes me a bit nervous, but I'm reluctant to expand our variant stored here all the way to a u32 without a good reason.

Reply:
Yeah, same reaction here.  My thoughts were it was worth trying the compatible thing first, and if it is a problem we can always change the public in the future if need be.